docs/sources/flow/reference/components/otelcol.exporter.jaeger.md

```diff
@@ -62,7 +62,7 @@ The following arguments are supported:
 
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
-`endpoint` | `string` | `host:port` to send telemetry data to. | | yes
+`endpoint` | `string` | `host:port` to send telemetry data to. | `""` | no
```

Comment:
I don't like this. IF we can avoid it while keeping the code clean, I'd prefer it. But not sure what is the best way? Copying the GRPCClientArguments struct seems like the simplest way to go.

Reply:
Yeah, I would personally copy the GRPCClientArguments for now until a better solution shows up; I don't think marking them all as optional for this one component should be the path we take.